modules/swagger-codegen/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -44,6 +44,10 @@
             <artifactId>spring-boot-starter-tomcat</artifactId>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
@cbornet if you want you can to directly modify the branch of my git repository (access contributor)